allennlp/data/dataset_readers/brown_corpus.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict
+import logging
+
+from overrides import overrides
+import tqdm
+
+from allennlp.common import Params
+from allennlp.common.checks import ConfigurationError
+from allennlp.data.dataset import Dataset
+from allennlp.data.dataset_readers.dataset_reader import DatasetReader
+from allennlp.data.instance import Instance
+from allennlp.data.token_indexers.token_indexer import TokenIndexer
+from allennlp.data.fields import TextField, TagField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+@DatasetReader.register("brown_corpus")
+class BrownCorpusDatasetReader(DatasetReader):
+    """
+    Reads instances from the Brown Corpus where each line is in the following format:
+
+    WORD/TAG WORD/TAG WORD/TAG ... WORD/TAG
```

Comment:
What if `WORD` or `TAG` has a `/`?  I think the logic below needs to do a `pair.rsplit("/", 1)`, and this needs a comment that `/` is not allowed as a character in `TAG`.